doc/release_notes_1.1.rst

```diff
@@ -0,0 +1,519 @@
+
+   **Impact:** Cannot use multiple USB devices at same time.
+
+   **Workaround:** Unplug and plug-in the unrecognized device after booting.
+
+-----
+
+:acrn-issue:`1991` - Input not accepted in UART Console for corner case
+   Input is useless in UART Console for a corner case, demonstrated with these steps:
+
+   1) Boot to SOS
+   2) ssh into the SOS.
+   3) use ``./launch_UOS.sh`` to boot UOS.
+   4) On the host, use ``minicom -s dev/ttyUSB0``.
+   5) Use ``sos_console 0`` to launch SOS.
+
+   **Impact:** Fails to use UART for input.
+
+   **Workaround:** Enter other keys before typing :kbd:`Enter`.
+
+-----
+
+:acrn-issue:`2267` - [APLUP2][LaaG] LaaG can't detect 4k monitor
+   After launching UOS on APL UP2 , 4k monitor cannot be detected.
+
+   **Impact:** UOS can't display on a 4k monitor.
+
+   **Workaround:** Use a monitor with less than 4k resolution.
+
+-----
+
+:acrn-issue:`2279` - [APLNUC] After exiting UOS, SOS can't use USB keyboard and mouse
+   After exiting UOS with mediator Usb_KeyBoard and Mouse, SOS cannot use the USB keyboard and mouse.
+
+   These steps reproduce the issue:
+
+   1) Insert USB keyboard and mouse in standard A port (USB3.0 port)
+   2) Boot UOS by sharing the USB keyboard and mouse in cmd line:
+
+      ``-s n,xhci,1-1:1-2:1-3:1-4:2-1:2-2:2-3:2-4 \``
+
+   3) UOS access USB keyboard and mouse.
+   4) Exit UOS.
+   5) SOS tries to access USB keyboard and mouse, and fails.
+
+   **Impact:** SOS cannot use USB keyboard and mouse in such case.
+
+   **Workaround:** Unplug and plug-in the USB keyboard and mouse after exiting UOS.
+
+-----
+
+:acrn-issue:`2753` - UOS cannot resume after suspend by pressing power key
+   UOS cannot resume after suspend by pressing power key
+
+   **Impact:** UOS may failed to resume after suspend by pressing the power key.
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`2974` - Launching Zephyr RTOS as a real-time UOS takes too long
+   Launching Zephyr RTOS as a real-time UOS takes too long
+
+   These steps reproduce the issue:
+
+   1) Build Zephyr image by follow the `guide
+      <https://projectacrn.github.io/latest/tutorials/using_zephyr_as_uos.html?highlight=zephyr>`_.
+   2) Copy the "Zephyr.img", "OVMF.fd" and "launch_zephyr.sh" to ISD.
+   3) execute the launch_zephyr.sh script.
+
+   **Impact:** Launching Zephyr RTOS as a real-time UOS takes too long
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`3268` - dm: add virtio-rnd device to command line
+   LaaG's network is unreachable with uos kernel
+
+   These steps reproduce the issue:
+
+   1) Downoad clearlinux
+   2) Decompress clearlinux
+   3) Replace above kvm.img with default kernel in uos
+   4) Launch UOS
+   5) Try to ping uos from other host.
+   6) UOS network is unreachalbe.
+
+   **Impact:** LaaG's network is unreachable with uos kernel
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`3281` - AcrnGT emulation thread causes high cpu usage when shadowing ppgtt
```

Comment:
```suggestion
:acrn-issue:`3281` - AcrnGT emulation thread causes high CPU usage when shadowing ``ppgtt``
```

It seems wrong that the summary, detail, and impact all say the exact same thing in these open issues.

Reply:
We would like to remove this known issue `3281` as the reason was given below:
> In current design, when guest updating its PPGTT table, GVT-g may call gfn_to_mfn frequently. So there's some performance improvement space by optimizing the gfn_to_mfn function call.
However, this commit is not well designed, and we may refactor the VHM module later. So this commit can be dropped. And we can close this issue as "won't fix" since we will have better designed solution in the near future.